WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -473,6 +473,21 @@ function WeakAuras.regionPrototype.AddExpandFunction(data, region, cloneId, pare
     WeakAuras.Animate("display", data, "main", data.animation.main, region, false, nil, true, cloneId);
   end
 
+  function region:OptionsClosed()
+    region:EnableMouse(false)
+    region:SetScript("OnMouseDown", nil)
+  end
+
+  function region:ClickToPick()
```

Comment:
Where's the call to ClickToPick for existing buttons? Is that not needed?

Reply:
I'm not sure if i understand your question.
region:ClickToPick() is called by region:Expand() if WeakAuras.IsOptionsOpen()
When clicking on an aura button or the visibility toggle, ClickToPick() is called by region:Expand()